bin/reth/src/main.rs

```diff
@@ -12,44 +11,58 @@ use reth_node_builder::{NodeBuilder, NodeConfig, NodeHandle};
 use reth_node_ethereum::EthereumNode;
 use reth_tasks::TaskManager;
 
+const BASE_CHAIN_ID: u64 = gwyneth::exex::BASE_CHAIN_ID; // Base chain ID for L2s
+const NUM_L2_CHAINS: u64 = 2; // Number of L2 chains to create
```

Comment:
Goo for now, but will have to be an array of specific chain_ids in the future coming from a config file.